tests/test_api_gen.py

```diff
@@ -85,9 +85,26 @@ def test_get_attributes(token, client, schemas_simple):
                 ["b_number", "float", False, None, False],
             ],
         },
+        "unique_values": [
+            [{"text": "0", "value": 0}, {"text": "1", "value": 1}],
+            [
+                {"text": "10", "value": 10},
+                {"text": "11", "value": 11},
+                {"text": "21", "value": 21},
+            ],
+            [
+                {"text": "Raphael", "value": "Raphael"},
+                {"text": "Bernie", "value": "Bernie"},
+            ],
+            [
+                {"text": "22.12", "value": 22.12},
+                {"text": "-1.21", "value": -1.21},
+                {"text": "7.77", "value": 7.77},
+            ],
+        ],
     }
 
-    assert expected_json == REST_response.get_json()
+    assert expected_json == REST_response.get_json(force=True)
```

Comment:
Why was `force` needed?

Reply:
I do not think it is needed so i removed it.

Reply:
looked into it and it seems that without `force=True` the method returns `none` which breaks the test.

Reply:
Looks like the API endpoint is not setting the [MIME](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) type for some reason e.g. `Content-Type: application/json`. We should fix that instead of setting it with [force](https://tedboy.github.io/flask/generated/generated/flask.Request.get_json.html).

Also, from [here](https://werkzeug.palletsprojects.com/en/2.2.x/test/#request-body) we might be able to use the smaller form:

```suggestion
    assert expected_json == REST_response.json()
```